common/views/components/ContentPage/ContentPage.js

```diff
@@ -144,6 +145,14 @@ const ContentPage = ({
               </Layout8>
             </SpacingSection>
           )}
+
+          {id === 'X5amzBIAAB0Aq6Gm' && (
```

Comment:
can we not create a util that can find hardcoded prismic ids ? my concern is having id's everywhere in the code base.
Also putting the id's in one place so we can remove or update easier..